### ISSUE TITLE
Metric counts in MetadataView failing to display in Edge

### ISSUE THREAD

**Issue by laurenwalker** [NCEAS/metacatui#934]
I noticed that the metric count is failing to display on search.dataone.org on the MetadataView. The exclamation point error icon is displayed instead. It works fine in all other browsers, even IE 11, oddly. The metrics are displaying okay on the DataCatalogView in Edge.

<img width="1051" alt="Screen Shot 2019-03-19 at 3 31 10 PM" src="https://user-images.githubusercontent.com/6643222/54636277-76ace600-4a5c-11e9-8821-5b9bc48e5ca1.png">

Metrics network request:
<img width="975" alt="Screen Shot 2019-03-19 at 3 34 03 PM" src="https://user-images.githubusercontent.com/6643222/54636276-76ace600-4a5c-11e9-8f17-8778f283fceb.png">

OS: Window 10
Browser: Edge

**Comment by laurenwalker** [comment]
Actually this doesn't work sometimes in IE 11 either. Sometimes when I reload the page it displays and sometimes it doesn't...

**Comment by rushirajnenuji** [comment]
Oh, this is interesting! I was not able to regenerate the issue - from NCEAS Viz Lab on Windows 10. Tested 10 different datasets from Data Catalog View and all 10 displayed the counts:
- Edge Version: ` 42.17134.1.0`
- IE 11 Version: `11.590.17134.0`

![IE11](https://user-images.githubusercontent.com/19696935/54639008-9c7cbf80-4a49-11e9-9dff-ce04f237beb2.PNG)
![Edge](https://user-images.githubusercontent.com/19696935/54639013-9edf1980-4a49-11e9-9adf-9e5a485b5ecf.PNG)

@laurenwalker - could you please specify your browser versions? It might help me regenerate the issue. Thanks!

**Comment by amoeba** [comment]
Just adding a data point here: I can't reproduce this on IE 11.379.1177763.0 or Edge 44.17763.1.0.

**Comment by laurenwalker** [comment]
I am on Edge 20. I looked at our Google Analytics and all our Edge users are on versions 15-17. (Yikes!)

I debugged this some more and it seems like an Edge issue that we might not be able to fix on our end. The XHR response status is `0`, which triggers an error, even though when I look at the network request in the developer tools, the status code is shown as a `200`. This seems like an issue other people are having as well: https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/14407632/

I'll go ahead and close this since it looks like there is nothing we can do
